how-to/running.rst

```diff
@@ -417,6 +419,23 @@ the bridge interface on your computer):
     sudo ufw allow in on lxdbr0
     sudo ufw route allow in on lxdbr0
 
+.. _building-on-arm:
+
+Building on ARM
+---------------
+
+When running ``make schema``, installing ``pynacl`` can run into `a bug <https://github.com/pyca/pynacl/issues/553>`_ when building for an ARM architecture, leading to ``ERROR: Failed building wheel for pynacl``.
+
+The following needs to be run when having this issue:
+
+.. code-block:: shell-session
+
+    $ sudo apt install --yes libsodium-dev 
+    $ source env/bin/activate
+    $ SODIUM_INSTALL=system pip install --no-binary :all: pynacl
```

Comment:
I'd prefer it if the instructions can be updated to build the local `sdist` of `PyNaCl` using `$(PIP)` instead of `pip install`'ing it from the internet. If the version of `PyNaCl` in the `lp-source-dependencies` doesn't work, we could add an `sdist` for the earliest (not the latest) version that works.

I'd also love to see a full test run passing on arm64 with this fix before documenting it.

Reply:
Is there a similar place I should add deb dependencies? Since I'd need to also add `libsodium-dev`

Reply:
Never mind, found it in `rocketfuel-setup`

Reply:
I put instructions to add the build flag to the makefile so this is not an issue.

Reply:
Can we not `pip install` things from the internet without any pinning and instead install a pinned version from the `sdist` file in the `lp-source-dependencies` like we do for all other dependencies?

Reply:
It is installing the same version it was before now, just adding to the flags. This diff is outdated